src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php

```diff
@@ -84,13 +85,14 @@ protected function configure()
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $documentRoot = $input->getOption('docroot');
+        $outputStyle = new SymfonyStyle($input, $output);
```

Comment:
Yes I understand, the var name in this cmd is different because $output is used here https://github.com/94noni/symfony/blob/14138_commands_style/src/Symfony/Bundle/FrameworkBundle/Command/ServerRunCommand.php#L125

ok for waiting a proper name